app/objects/c_operation.py

```diff
@@ -57,7 +56,7 @@ def states(self):
                     FINISHED='finished',
                     CLEANUP='cleanup')
 
-    def __init__(self, name, agents, adversary, id=None, jitter='2/8', source=None, planner=None, state='running',
+    def __init__(self, name, agents, adversary, id='', jitter='2/8', source=None, planner=None, state='running',
```

Comment:
In the `__init__` method itself, should we cast `self.id = str(id)`? I'm wondering if there would be any issues if someone passed in an integer.